vnext/Microsoft.ReactNative/pch.h

```diff
@@ -1,4 +1,3 @@
-#pragma once
```

Comment:
>pragma [](start = 1, length = 6)

why is this removed?

Reply:
It was not intentional - I am going to revert it.